src/client.ts

```diff
@@ -321,9 +326,88 @@ export class OriginalClient {
   public async getDeposit(userUid: string) {
     return await this._get<APIResponse<Deposit>>('deposit', { user_uid: userUid });
   }
+
+  /**
+   * reward methods
+   */
+
+  /**
+   * getReward
+   *
+   * @param {String} uid Uid of the reward to get
+   * @return {Promise<APIResponse<Reward>>} Returns the details of the reward.
+   * Will throw a 404 error if the reward does not exist.
+   */
+  public async getReward(uid: string) {
+    return this._get<APIResponse<Reward>>(`reward/${uid}`);
+  }
+
+  /**
+   * allocation methods
+   */
+
+  /**
+   * createAllocation
+   * @param {AllocationParams} allocation The details of the allocation to be created
+   * @return {Promise<APIResponse<UidResponse>>} Uid of the created allocation
+   */
+  public async createAllocation(allocation: AllocationParams) {
+    return await this._post<APIResponse<UidResponse>>('reward/allocate', allocation);
```

Comment:
No need to `return await` in JS as we return a promise. 
We can just do:
`return this._post<APIResponse<UidResponse>>('reward/allocate', allocation);`

Reply:
thanks, I will also remove the unnecessary await on Burn then